pkg/apitype/stacks.go

```diff
@@ -53,6 +56,11 @@ type EncryptValueRequest struct {
 	Plaintext []byte `json:"plaintext"`
 }
 
+type CurrentUpdateStatus struct {
+	CurrentUpdateStarted int64 `json:"currentUpdateStarted,omitempty"`
+	CurrentUpdateAuthor string `json:"currentUpdateAuthor,omitempty"`
```

Comment:
It's rare that you should ever include the type name as a prefix in one of its fields. This is referred to as "stuttering".

In this case, the field should probably just be named `Author` instead of `CurrentUpdateAuthor`. (Since the "CurrentUpdate" prefix is just duplication from the type's name, and would apply to every other field in the type.)